### PR TITLE
[16.0][MIG] hr_attendance_rfid: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT45866

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
Code review :smile:

File: hr_attendance_rfid/security/hr_attendance_rfid.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_hr_attendance_rfid" model="res.groups">
+        <field name="name">RFID Attendance</field>
+        <field
+            name="category_id"
+            ref="base.module_category_human_resources_attendances"
```
Comment (pedrobaeza) [inline hr_attendance_rfid/security/hr_attendance_rfid.xml]: Putting into this category breaks the dropdown, so it should be in another like technical one.

(It's not a migration issue, but something I have found).

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**